### PR TITLE
[PM-3681] Remove ipcRenderer from lock.component

### PR DESCRIPTION
**Pull request by dani-garcia**
## Type of change
```
- [ ] Bug fix
- [ ] New feature development
- [x] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective
Removed all uses of `ipcRenderer` from lock.component.ts, and moves them to use the ipc preload, getting us step closer to enabling electron sandboxing. This depends on #6065.

### CODE REVIEW

**Review by JaredSnider-Bitwarden — DISMISSED** [review]
LGTM!

File: apps/desktop/src/platform/preload.ts [hunk 1]
```diff
@@ -12,6 +13,9 @@ export default {
   isDev: isDev(),
   isWindowsStore: isWindowsStore(),
   reloadProcess: () => ipcRenderer.send("reload-process"),
+
+  isWindowVisible: (): Promise<boolean> => ipcRenderer.invoke("windowVisible"),
+  biometric: (message: BiometricMessage) => ipcRenderer.invoke("biometric", message),
```
[the remark this answers is not in the document]
Comment (dani-garcia) [reply apps/desktop/src/platform/preload.ts]: That's a nice idea yeah, it makes the code a bit bigger, so I've defined separately from the default export block to not clutter it too much.

**Review by Hinton — DISMISSED** [review]

**Review by JaredSnider-Bitwarden — DISMISSED** [review]
Nice change!

**Review by Hinton — APPROVED** [review]

**Review by Hinton — APPROVED** [review]

**Review by JaredSnider-Bitwarden — APPROVED** [review]
👍

**Pull request merged**